projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -131,35 +142,35 @@
   "ars_nouveau.spell.disabled": "该法术已在配置中被禁用。",
-  "block.ars_nouveau.sourceberry_bush": "魔源浆果丛",
+  "block.ars_nouveau.sourceberry_bush": "魔源浆果",
   "ars_nouveau.connections.cleared": "连接已清除。",
-  "ars_nouveau.connections.take": "中继器已设置成从%s抽取魔力",
-  "ars_nouveau.connections.send": "中继器已设置成将魔力传输至%s",
-  "item.minecraft.splash_potion.effect.mana_regen_potion_strong": "喷溅型魔力再生药水",
-  "item.minecraft.splash_potion.effect.mana_regen_potion": "喷溅型魔力再生药水",
-  "item.minecraft.splash_potion.effect.mana_regen_potion_long": "喷溅型魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion_strong": "魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion": "魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion_long": "魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion_strong": "滞留型魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion": "滞留型魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion_long": "滞留型魔力再生药水",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion_strong": "魔力再生之箭",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion": "魔力再生之箭",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion_long": "魔力再生之箭",
-  "effect.ars_nouveau.mana_regen": "魔力再生",
+  "ars_nouveau.connections.take": "中继器将从%s抽取魔力",
+  "ars_nouveau.connections.send": "中继器会将魔力传输至%s",
+  "item.minecraft.splash_potion.effect.mana_regen_potion_strong": "喷溅型魔力恢复药水",
+  "item.minecraft.splash_potion.effect.mana_regen_potion": "喷溅型魔力恢复药水",
+  "item.minecraft.splash_potion.effect.mana_regen_potion_long": "喷溅型魔力恢复药水",
+  "item.minecraft.potion.effect.mana_regen_potion_strong": "魔力恢复药水",
+  "item.minecraft.potion.effect.mana_regen_potion": "魔力恢复药水",
+  "item.minecraft.potion.effect.mana_regen_potion_long": "魔力恢复药水",
+  "item.minecraft.lingering_potion.effect.mana_regen_potion_strong": "滞留型魔力恢复药水",
+  "item.minecraft.lingering_potion.effect.mana_regen_potion": "滞留型魔力恢复药水",
+  "item.minecraft.lingering_potion.effect.mana_regen_potion_long": "滞留型魔力恢复药水",
+  "item.minecraft.tipped_arrow.effect.mana_regen_potion_strong": "魔力恢复之箭",
+  "item.minecraft.tipped_arrow.effect.mana_regen_potion": "魔力恢复之箭",
+  "item.minecraft.tipped_arrow.effect.mana_regen_potion_long": "魔力恢复之箭",
+  "effect.ars_nouveau.mana_regen": "魔力恢复",
   "ars_nouveau.on": "激活：是",
   "ars_nouveau.off": "激活：否",
   "item.ars_nouveau.void_jar": "湮灭之瓶",
-  "ars_nouveau.rune.setperm": "这个符文现在是永久性的了。",
-  "ars_nouveau.page.void_jar": "一个可以销毁拾取的物品，并从销毁中产生魔力的瓶子。潜行使用可打开或关闭瓶子。要添加或移除瓶子能销毁的物品，可在将物品拿在副手时使用瓶子，也可以把瓶子放在抄写台上，再持物品对抄写台使用。湮灭之瓶必须放在快捷栏才能工作。",
-  "ars_nouveau.page.runic_chalk": "符文粉笔可用于将羊皮纸符文放在地上，这种符文会对走过的实体施放法术。要给符文附上法术，只需对其使用在抄写台抄写的法术羊皮纸。一旦符文施放了一个法术，它会变成未充能状态。未充能的符文会自动从附近的魔源罐抽取魔源进行充能。在临时符文上使用符文粉笔可以将其转换为永久符文。",
-  "ars_nouveau.wand.invalid": "无效法术。法杖的法术只能含效果类和增强类构件。",
+  "ars_nouveau.rune.setperm": "符文已变为永久符文。",
+  "ars_nouveau.page.void_jar": "一个可以销毁拾取的物品并产生魔力的瓶子，潜行右键可打开或关闭销毁功能。在主手使用湮灭之瓶可以将副手物品加入或移出白名单，对抄写台上的湮灭之瓶使用物品亦可。湮灭之瓶只能在快捷栏中生效。",

```

`右击`是动词，`右键`是名词，这里用`右击`合适
```suggestion
  "ars_nouveau.page.void_jar": "一个可以销毁拾取的物品并产生魔力的瓶子，潜行右击可打开或关闭销毁功能。在主手使用湮灭之瓶可以将副手物品加入或移出白名单，对抄写台上的湮灭之瓶使用物品亦可。湮灭之瓶只能在快捷栏中生效。",
```

---

个人习惯是“右键使用”“对方块右击使用”，右键用于直接使用物品的场景，右击使用于对特定目标使用物品的场景